### ISSUE TITLE
Mark this project as deprecated!

### ISSUE THREAD

**Issue by dkern** [adldap/adLDAP#144]
It seems this project is dead since a long time. Your docs are still talking about the "next weeks" but has no commit, change or merge since years. As there seems to be no change or contribute anymore at least tell other devs about it ...

**Comment by strebl** [comment]
@eisbehr- go and use https://github.com/Adldap2/Adldap2 it‘s the successor of this package.

**Comment by dkern** [comment]
@strebl Yes, I know. A reason more to close this repo here or at least leave a mark. But I think nobody else cares. ;)
